widgetssdk/src/main/java/com/glia/widgets/chat/ChatContract.kt

```diff
@@ -82,5 +83,6 @@ internal interface ChatContract {
         fun showToast(message: String, duration: Int = Toast.LENGTH_SHORT)
         fun dispatchImageCapture(uri: Uri)
         fun onFileDownload(attachmentFile: AttachmentFile)
+        fun navigateToPreview(attachmentFile: LocalAttachment, view: android.view.View)
```

Comment:
navigateToPreview -> navigateToFilePreview ?
I am a bit picky about naming )
But if you disagree with any of those then please share your thoughts I am happy to discuss and change my mind

Reply:
renamed - c64be489fee6ea4819df6498f1bd8e2ab9977e99